Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,11 +4,68 @@
 using Console = System.Console;
 
 namespace BootCamp.Chapter
-{
+{ 
     class Program
     {
         static void Main(string[] args)
         {
+            string nameMessage = "May I have your name?";
+            string surnameMessage = "May I have your surname?";
+            string ageMessage = "May I have your age?";
+            string weightMessage = "May I have your weight(KG)?";
+            string heightMessage = "May I have your height(m)?";
+
+            string name = AskingAndConvertingSTRING(nameMessage);
+            string surname = AskingAndConvertingSTRING(surnameMessage);
+            int age = AskingAndConvertingINT(ageMessage);
+            float weight = AskingAndConvertingFLOAT(weightMessage);
+            float height = AskingAndConvertingFLOAT(heightMessage);
+
+            Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+            float bmi = CalculateBMI(weight, height);
+
+            Console.WriteLine("His BMI is " + bmi + ".");
+
+            // Up - first person, down - second.
+
+            string secondName = AskingAndConvertingSTRING(nameMessage);
+            string secondSurname = AskingAndConvertingSTRING(surnameMessage);
+            int secondAge = AskingAndConvertingINT(ageMessage);
+            float secondWeight = AskingAndConvertingFLOAT(weightMessage);
+            float secondHeight = AskingAndConvertingFLOAT(heightMessage);
+
+            Console.WriteLine(secondName + " " + secondSurname + " is " + secondAge + " years old, his weight is " + secondWeight + " kg and his height is " + secondHeight + " cm.");
+
+            float secondBmi = CalculateBMI(secondWeight, secondHeight);
+
+            Console.WriteLine("His BMI is " + secondBmi + ".");
+
+            Console.ReadLine();
+
+        }
+        public static float CalculateBMI(float weight, float height)
+        {
+            float bmi = weight / (height * height);
+            return (float)Math.Round(bmi, 1);
+        }
+        public static int AskingAndConvertingINT(string message)
+        {
+            Console.WriteLine(message);
+            int integer = Int32.TryParse(Console.ReadLine(), 10);
```

Comment:
If you looked at the error messages:
![image](https://user-images.githubusercontent.com/40486932/96492380-3b802380-124c-11eb-962e-0ad9648a7525.png)
It shows what's wrong.

And indeed, that's not how TryParse worse. Please note, that TryParse returns true or false (was parse successful).
It also returns the parsed value, but that is done using the `out` keyword.

What you actually need is this:
```
bool isNumber = Int32.TryParse(Console.ReadLine(), out int number);
```
or if you trust that it's always going to be a numberr, you can simply do this:
```
int number = int.Parse(Console.ReadLine());
```